ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -2475,3 +2475,35 @@ def test_tensorflow_greater_equal(
         x=x[0],
         y=x[1],
     )
+    
+    
+    
+    
+# in_top_k
+@handle_frontend_test(
+    fn_tree="tensorflow.math.in_top_k",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_in_top_k(
+    *,
+    dtype_and_x,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
```

Comment:
Please pass the values to `target`, `pred`, and `k`.

Reply:
How ??

Reply:
You can go through other testing functions in the same file and see how that's done, you can also go through the docs.